packages/ui5-info/src/ui5-version-info.ts

```diff
@@ -343,33 +343,30 @@ export async function getUI5Versions(filterOptions?: UI5VersionFilterOptions): P
         );
         filteredUI5Versions = defaultUi5Versions.slice();
     }
+
     const defaultUI5Version = filteredUI5Versions[0];
+    const ui5VersionsOverview = filterOptions?.includeMaintained
```

Comment:
Not sure we need a ternary operator here?

Reply:
Addressed, with some additional comments, thanks!